src/pages/Mypage/Mypage.js

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+export default function MyPage() {
+  const [guestData, setGuestData] = useState([]);
+  const [hostData, setHostData] = useState([]);
+
+  useEffect(() => {
+    getGuestData();
+  }, []);
+
+  useEffect(() => {
+    getHostData();
+  }, []);
```

Comment:
하나의 useEffect로 작성해주세요!
```suggestion
  useEffect(() => {
    getGuestData();
    getHostData();
  }, []);
```

Reply:
하나로 합쳤습니다~!